src/css/vendor/_variables_uswds.scss

```diff
@@ -6,7 +6,7 @@
 // Removing the !default from $em-base so we are not inheriting that
 // value from Bourbon.
 
-@import url('https://fonts.googleapis.com/css?family=Montserrat:400,500,700');
+@import url('https://fonts.googleapis.com/css?family=Lora:400,700|Open+Sans:400,700');
```

Comment:
I defined the full font set based on the stickersheet as 
`@import url('https://fonts.googleapis.com/css?family=Lora|Open+Sans:400,600,700,800');`
in _variables_coa.scss
https://github.com/cityofaustin/janis/blob/fb35ba5e62f834729f833345dfb06d113559e5d1/src/css/base/_variables_coa.scss